### PR TITLE
authentication complete

### PR DESCRIPTION
**Pull request by JINDALG**
login and signup process complete.

### CODE REVIEW

File: myresume/templates/account.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <div class="container">
+      <div style="margin-top:50px"class="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
+        <div ng-show="showing" class="panel panel-info">
+            <div class="panel-heading">
+             <div class="panel-title">Log In</div>
+             <div style="float:right; font-size: 80%; position: relative; top:-10px"><a href="#">Forgot password?</a></div>
+            </div>
+            <div  class="panel-body">
+            <form id="login" onsubmit="log()">
+            {% csrf_token %}
```
Comment (aayusharora) [inline myresume/templates/account.html]: @JINDALG , why do we have Django variables here , can login be done by Rest API .

**Comment by aayusharora** [comment]
@JINDALG  Awesome Please see the comment

File: myresume/templates/account.html (hunk 1 above)
Comment (JINDALG) [inline myresume/templates/account.html]: {% csrf_token %} is for prevention from SQL injection

**Comment by JINDALG** [comment]
Database structure is defined. Please check it and comment if you find something need to be correct.

**Comment by aakarshgoel96** [comment]
@JINDALG  database structure seems ok, just wanna know that this database can store multiple project and internship objects or not??

**Comment by JINDALG** [comment]
It can store multiple entity of projects and internship as well as activity

**Comment by aakarshgoel96** [comment]
ok then great @JINDALG

**Pull request merged**